docs/addons/addon-migration-guide.md

```diff
@@ -34,6 +34,15 @@ Please upgrade your addon to Node.js 18, as support for Node.js 16 has ended.
 
 UI injected into panels, tools, etc. by addons is now rendered with React 18. Also note that the `key` prop is no longer passed to the render function.
 
+### React peer dependency is no longer required
```

Comment:
@vanessayuenn, when you have a moment, can you adjust this section to include a mention for users who are using an outdated version of the addon kit that was pre-tsup to reach out to us via our communication channels (e.g., GitHub discussions) as we don't have a direct path for such cases?

Reply:
see this [comment](https://github.com/storybookjs/storybook/pull/25826/files#r1488518864)